src/modules/logger/logger.cpp

```diff
@@ -525,14 +525,14 @@ void Logger::add_default_topics()
 	add_topic("manual_control_setpoint", 200);
 	add_topic("mission");
 	add_topic("mission_result");
-	add_topic("optical_flow", 50);
-	add_topic("position_controller_status", 500);
+	add_topic("position_controller_status", 1000);
 	add_topic("position_setpoint_triplet", 200);
 	add_topic("radio_status");
 	add_topic("rate_ctrl_status", 200);
 	add_topic("sensor_combined", 100);
 	add_topic("sensor_preflight", 200);
-	add_topic("system_power", 500);
```

Comment:
Without System power at 2 Hz (or more) you can hardly debug power-loss scenarios, of which I've seen now a few in my career. Because its the only straw to tell it apart from a hardfault (apart from the hard fault log) I think we need to be diligent here.

Reply:
I'll leave it at 2 Hz.